core-configs/.celestia-app-10/config/config.toml

```diff
@@ -463,7 +463,7 @@ timeout-precommit-delta = "500ms"
 # How long we wait after committing a block, before starting on the new
 # height (this gives us a chance to receive some more precommits, even
 # though we already have +2/3).
-timeout-commit = "1s"
+timeout-commit = "25s"
```

Comment:
Did you manually bump those or did you change this in a master config or in the code somewhere?

Reply:
I've manually bumped them to test out locally. This will be adjusted to be automatic through code later (we already have automatic assignment of persistent-peers, so this should be def easy job to do)